src/steps/10-FinancialDetails/IGCE/CostSummary.vue

```diff
@@ -264,7 +377,22 @@ export default class CostSummary extends Vue {
     if(ditcoFee || contratingFee){
       this.createFeeData("grandTotal", grandTotal, "false")
     }
-    // this.createTableData(ditcoFee,"false","Ditco FEE")
+    await this.findMissingEstimates()
+  }
+  public editRoute():void {
+    let name = routeNames.GatherPriceEstimates
+    if(this.hasCurrentEnv){
+      name = routeNames.OptimizeOrReplicate
+    }
+    if(this.hasArchDesign){
```

Comment:
pls change this back to an `else if` -- OptimizeOrReplicate is first in the stepper so should go there if `hasCurrentEnv` - without the `else` if they do also have arch design, they wouldn't get directly to that first OptimizeOrReplicate page